gnomad_constraint/pipeline/constraint_pipeline.py

```diff
@@ -392,6 +455,47 @@ def main(args):
         ),
         action="store_true",
     )
+    parser.add_argument(
+        "--calculate-mutation-rate",
+        help="Calculate mutation rate",
```

Comment:
Maybe add more info, for instance the resource says "Return mutation Table that includes the baseline mutation rate for each substitution and context"